src/core/Buttons/Button/Button.tsx

```diff
@@ -5,10 +5,10 @@
 import cx from 'classnames';
 import React from 'react';
 
-import { useTheme } from '../../utils';
+import { PolymorphicComponentPropsWithRef, useTheme } from '../../utils';
```

Comment:
Add a story with button as an anchor.

Reply:
Read PR description. Would be weird to publish a story with a visual bug

Also rendering button as anchor is a bad practice so I'm not even sure we want to make it obvious that it's possible

Reply:
Link to bad practice?

Reply:
There are two problems:
- screen readers will announce button as a link.
- clicking spacebar on a link scrolls the page, so it will confuse even normal users.

https://marcysutton.com/links-vs-buttons-in-modern-web-applications

Of course there can be some valid use cases like a download link or a "get started" link, but those are less common.

Reply:
What about buttons in header, which navigate you to some page? You want them to be anchors, to support middle clicka nd show url.. but you need them to look as buttons.

Reply:
not sure we want to use our normal buttons for those. But like I said, there are some valid use cases. It's just that if we advertise this feature, it has a great potential for misuse.

Reply:
But if we don't show it, then it will be missed in dozens of native props.

Reply:
I would say lets add a story so users are aware that if there is a need, we support that. .otherwise, it is not that obvious to even try to find it. 
And we can complete this PR.

Reply:
Added story